tests/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+
+set("TEST_PG_DIR" "" CACHE STRING "Postgres installation directory used for testing")
+
+if("${TEST_PG_DIR}" STREQUAL "")
+    message("TEST_PG_DIR is empty, trying to autodetect")
+    set(POSSIBLE_PG_VERS 17 16 15 14)
+    foreach(ver ${POSSIBLE_PG_VERS})
+        set(dir "/usr/lib/postgresql/${ver}")
+        if(EXISTS "${dir}")
+            message("Setting TEST_PG_DIR to ${dir}")
+            set("TEST_PG_DIR" "${dir}")
+            break()
+        endif()
+    endforeach()
+endif()
```

Comment:
Not a blocker for the PR but wouldn't a potentially better solution be to call `pg_config --bindir`?

Reply:
I add this to the todo list, as it needs a bit more work - currently the assumption is that we want the root directory of pg, one above the binary dir, and there's no option for that in the bindir command. But in reality we only need the bindir.